src/ptf/mask.py

```diff
@@ -56,8 +62,17 @@ def pkt_match(self, pkt):
 
     def __str__(self):
         assert(self.valid)
-        return ''.join([chr(b) for b in self.mask])
-
+        old_stdout = sys.stdout
+        sys.stdout = buffer = StringIO()
+        hexdump(self.exp_pkt)
+        print 'mask =',
+        for i in range(0, len(self.mask), 16):
```

Comment:
wouldn't it be simpler and more "uniform" to just call `hexdump` on the mask?

Reply:
scapy.utils.hexdump prints the stringified format, not a list of integers. If there is a different hexdump that I can use, please give me a pointer.

Reply:
I see what you mean. I thought `mask` was a binary string, not an integer array...